enterprise_access/apps/api/serializers.py

```diff
@@ -141,7 +148,69 @@ def validate_content_key(self, value):
 
         return value
 
-class SubsidiyAccessPolicyListSerializer(SubsidiyAccessPolicyRedeemSerializer):  # pylint: disable=abstract-method
+
+class SubsidyAccessPolicyCRUDSerializer(serializers.ModelSerializer):
+    """
+    Serializer to validate policy data for CRUD operations.
+    """
+    uuid = serializers.UUIDField(read_only=True)
```

Comment:
Looking at this now. Does that mean `uuid` will not be used in any write operation?

Reply:
Yep, it will be auto-generated via [model](https://github.com/openedx/enterprise-access/blob/main/enterprise_access/apps/subsidy_access_policy/models.py#L37) and we would want to ensure nobody changes the value while updating the instance. so that's why we added `read-only`